lib/components/Information/Badge/index.tsx

```diff
@@ -26,6 +38,14 @@ export const Badge = forwardRef<HTMLDivElement, BadgeProps>(
             />
           </span>
         )}
+        {hasDot && (
+          <span
+            className={cn(
+              "mr-1 h-2 w-2 rounded-full",
+              variant && dotColor[variant]
```

Comment:
`variant` will always have a value right?
```suggestion
              dotColor[variant]
```